scripts/generate-links/str.mjs

```diff
@@ -0,0 +1,49 @@
+const PREPOSITIONS = [
```

Comment:
I think is ok to do this via code, but we should create a tech debt to pass this to a CI linter and force docs to use in the correct way.

Reply:
[FRO-609](https://linear.app/fuel-network/issue/FRO-609/replace-strts-with-a-linter)